src/command_modules/azure-cli-servicefabric/azure/cli/command_modules/servicefabric/custom.py

```diff
@@ -282,6 +284,17 @@ def new_cluster(client,
 
     return output_dict
 
+def _build_detailed_error(topError, outputList):
+    if outputList:
+        outputList.append(" Inner Error - Code: \"{}\" Message: \"{}\" \n".format(topError.code, topError.message))
+    else:
+        outputList.append("Error - Code: \"{}\" Message: \"{}\" \n".format(topError.code, topError.message))
```

Comment:
Can use single quotes for the string and then you don't need to escape the double quotes

Reply:
Great, done